src/pages/Account/Account.tsx

```diff
@@ -85,6 +111,13 @@ export function Account() {
     toggleAllTransactions()
   }
 
+  const handleTransactionFilter = useCallback(({ target: { name, value } }: ChangeEvent<HTMLSelectElement>) => {
+    setPage(1)
+    searchParams.set(name, value)
+    setSearchParams(searchParams, { replace: true })
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
```

Comment:
This is on purpose — run this effect just once.

Reply:
Yes, that's the purpose but looks that something is wrong. Using `// eslint-disable-next-line react-hooks/exhaustive-deps` too much is a code smell. I think it would be good to look into the code and try to remove all this comments.